dashboard/app/jobs/concerns/active_job_metrics.rb

```diff
@@ -148,9 +148,26 @@ def self.report_metrics(job_class, dimensions:)
   end
 
   def self.report_overall_queue_metrics
+    migrate_failed_jobs
     ActiveJobMetrics.report_metrics(ActiveJobMetrics, dimensions: [{name: 'Environment', value: CDO.rack_env}])
   end
 
+  def self.migrate_failed_jobs
+    return unless ActiveRecord::Base.connection.table_exists? 'failed_delayed_jobs'
```

Comment:
Can we just do:

```suggestion
    return unless FailedDelayedJob.table_exists?
```

?